hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/AbstractIntegrationTest.java

```diff
@@ -435,4 +436,22 @@ protected static String generateRandomStringWithLength(final int length) {
 
         return randomStringBuilder.toString();
     }
+
+
+    protected static Action getAssignedAction(final DistributionSetAssignmentResult distributionSetAssignmentResult) {
```

Comment:
Maybe call this method getFirstAssignedAction (also put "first" in the name of the other methods)? That way it is more clear that using this method in a test means that all other actions, that might be in the result, are ignored.